code/game/machinery/vending.dm

```diff
@@ -1143,6 +1143,52 @@
 	idle_power_usage = 211 //refrigerator - believe it or not, this is actually the average power consumption of a refrigerated vending machine according to NRCan.
 	auto_price = FALSE
 
+/obj/machinery/vending/medical/commercial
+	name = "MiniPharma Trade"
+	desc = "Medical drug dispenser."
+	icon_state = "med"
+	icon_deny = "med-deny"
+	req_access = list(access_medical_equip)
```

Comment:
Also why?

Reply:
Why have you put medical equipment access requirement? So med spec has to hand out pills that they bought with their own money? I don't really get it.

Reply:
👀

Reply:
It shouldn't have been there and it's no longer is. Thanks for pointing out.